flink-runtime/src/main/java/org/apache/flink/runtime/client/SerializedJobExecutionResult.java

```diff
@@ -63,6 +64,17 @@ public long getNetRuntime() {
 		return netRuntime;
 	}
 
+    /**
+     * Gets the net execution time of the job, i.e., the execution time in the parallel system,
+     * without the pre-flight steps like the optimizer in a desired unit.
+     *
+     * @param desiredUnit the unit of the <tt>NetRuntime</tt>
+     * @return The net execution time in the desired unit.
+     */
```

Comment:
The indentation of the comments above is done using spaces. We use tabs in Flink Java